src/_includes/base.njk

```diff
@@ -46,6 +46,18 @@
       function gtag(){dataLayer.push(arguments);}
       gtag('js', new Date());
       gtag('config', 'UA-57132539-4');
+
+      console.log(`%c
```

Comment:
It looks like you figured how the njk file works!

Can we do something less flashy? :)

Reply:
That's great!